### PR TITLE
[graphql] InstigationState lookup by selector fallback

### PR DESCRIPTION
**Pull request by alangenfeld**
If we are passed both id and selector, fallback to the selector if theres nothing in the DB for the ID

### CODE REVIEW

**Comment by alangenfeld** [comment]
* **#22974** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/22974?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @alangenfeld and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py [hunk 1]
```diff
@@ -729,20 +729,25 @@ def resolve_instigationStateOrError(
...
                 CompoundID.from_string(id),
+                selector,
             )
-        elif instigationSelector:
+        elif selector:
             return get_instigator_state_by_selector(
                 graphene_info,
-                InstigatorSelector.from_graphql_input(instigationSelector),
+                selector,
             )
```
Comment (salazarm) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py:748]: Should we just require both the ID and the selector ?

Comment (alangenfeld) [reply python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py:748]: yea could sort of revert back closer to how this used to be with id as an optional accelerant argument

Comment (salazarm) [reply python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py]: i like that framing

**Review by salazarm — APPROVED** [review]
LGTM

**Pull request merged**